web/src/components/MapAttribution.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { ReactElement, useState } from 'react'
+import styled from 'styled-components'
+
+import { openStreeMapCopyright } from 'api-client'
+
+import CleanLink from './CleanLink'
+
+const Attribution = styled.div`
+  display: flex;
+  padding: 0 4px;
+  background-color: rgba(255, 255, 255, 0.3);
+  box-shadow: 0 2px 3px 3px rgb(0 0 0 / 10%);
+  color: rgba(0, 0, 0, 0.75);
```

Comment:
Should be the same as in the footer overlay imo.
```suggestion
  background-color: rgba(255, 255, 255, 0.5);
  box-shadow: 0 2px 3px 3px rgba(0, 0, 0, 0.1);
  color: rgba(0, 0, 0, 0.75);
```